src/config.rs

```diff
@@ -23,34 +21,8 @@ impl Default for GlobalConfig {
 
 impl GlobalConfig {
     pub fn new() -> Self {
-        let is_stdout_tty = atty::is(atty::Stream::Stdout);
-        let is_stderr_tty = atty::is(atty::Stream::Stderr);
-
-        let color_choice = match std::env::var("CARGO_TERM_COLOR").as_deref() {
-            Ok("always") => Some(ColorChoice::Always),
-            Ok("alwaysansi") => Some(ColorChoice::AlwaysAnsi),
-            Ok("auto") => Some(ColorChoice::Auto),
-            Ok("never") => Some(ColorChoice::Never),
-            Ok(_) | Err(..) => None,
-        };
```

Comment:
`cargo` itself uses `CARGO_TERM_COLOR` and the `--color` flag: https://doc.rust-lang.org/cargo/reference/config.html#termcolor

As a cargo extension with plans to merge into cargo one day (#61), it would massively simplify things if we did the same thing. So I'd prefer to keep this mechanism available. (If there's no `alwaysansi` option in the new crate, that can go since cargo doesn't use it either.)

Reply:
I'd be a bit surprised if `anstyle` got first-party support for a cargo-specific way to configure color, but perhaps there's going to be an `anstyle-cargo` crate :) But I'm sure epage is super busy, and I feel reasonably confident it's safe to keep `CARGO_TERM_COLOR` support here unless he chimes in to say otherwise.